eng/Version.Details.xml

```diff
@@ -39,76 +39,76 @@
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Caching.StackExchangeRedis" Version="8.0.0">
-      <Uri>https://github.com/dotnet/aspnetcore</Uri>
-      <Sha>c838682d8984f0183abb48a4032bf7846fef52ae</Sha>
+    <Dependency Name="Microsoft.Extensions.Caching.StackExchangeRedis" Version="8.0.1">
+      <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-aspnetcore</Uri>
+      <Sha>02aeddda68390d089a8cdd32f5a25229cbe6daa3</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Diagnostics.HealthChecks.EntityFrameworkCore" Version="8.0.0">
-      <Uri>https://github.com/dotnet/aspnetcore</Uri>
-      <Sha>c838682d8984f0183abb48a4032bf7846fef52ae</Sha>
+    <Dependency Name="Microsoft.Extensions.Diagnostics.HealthChecks.EntityFrameworkCore" Version="8.0.1">
+      <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-aspnetcore</Uri>
+      <Sha>02aeddda68390d089a8cdd32f5a25229cbe6daa3</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Diagnostics.HealthChecks" Version="8.0.0">
-      <Uri>https://github.com/dotnet/aspnetcore</Uri>
-      <Sha>c838682d8984f0183abb48a4032bf7846fef52ae</Sha>
+    <Dependency Name="Microsoft.Extensions.Diagnostics.HealthChecks" Version="8.0.1">
+      <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-aspnetcore</Uri>
+      <Sha>02aeddda68390d089a8cdd32f5a25229cbe6daa3</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Features" Version="8.0.0">
-      <Uri>https://github.com/dotnet/aspnetcore</Uri>
-      <Sha>c838682d8984f0183abb48a4032bf7846fef52ae</Sha>
+    <Dependency Name="Microsoft.Extensions.Features" Version="8.0.1">
+      <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-aspnetcore</Uri>
+      <Sha>02aeddda68390d089a8cdd32f5a25229cbe6daa3</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.EntityFrameworkCore.Cosmos" Version="8.0.0">
-      <Uri>https://github.com/dotnet/efcore</Uri>
-      <Sha>e017dc125bef2f604f85befd8ff27544a5a67c38</Sha>
+    <Dependency Name="Microsoft.EntityFrameworkCore.Cosmos" Version="8.0.1">
+      <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-efcore</Uri>
+      <Sha>4c95cefe39426014c24cdaa8688518618bc0040f</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.EntityFrameworkCore.SqlServer" Version="8.0.0">
-      <Uri>https://github.com/dotnet/efcore</Uri>
-      <Sha>e017dc125bef2f604f85befd8ff27544a5a67c38</Sha>
+    <Dependency Name="Microsoft.EntityFrameworkCore.SqlServer" Version="8.0.1">
+      <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-efcore</Uri>
+      <Sha>4c95cefe39426014c24cdaa8688518618bc0040f</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.EntityFrameworkCore.Tools" Version="8.0.0">
-      <Uri>https://github.com/dotnet/efcore</Uri>
-      <Sha>e017dc125bef2f604f85befd8ff27544a5a67c38</Sha>
+    <Dependency Name="Microsoft.EntityFrameworkCore.Tools" Version="8.0.1">
+      <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-efcore</Uri>
```

Comment:
Can't the repo stay as the GH one as opposed to the AzDO one? The sha should be available in both.

Reply:
I'd prefer to keep it as-is since that's what the tooling will spit out. It shouldn't affect any dev workflow